sdk/communication/Azure.Communication.Rooms/README.md

```diff
@@ -52,8 +50,99 @@ RoomsClient client = new RoomsClient(connectionString);
 ```
 
 ## Examples
+### Create a room
+To create a room, call the `CreateRoom` or `CreateRoomAsync` function from `RoomsClient`.
+The `validFrom`, `validUntil`, and `participants` arguments are all optional. If `validFrom` and `validUntil` are not provided, then the default for `validFrom` is current date time and the default for `validUntil` is `validFrom + 180 days`.
+When defining `InvitedRoomParticipants`, if role is not specified, then it will be `Attendee` by default.
```

Comment:
Should this be 'InvitedRoomParticipant'?

Reply:
Yes, good catch, thank you!